pages/common/mysqlcheck.md

```diff
@@ -0,0 +1,20 @@
+# mysqlcheck
+
+> Check and repair MySQL tables.
+> More information: <https://dev.mysql.com/doc/refman/8.0/en/mysqlcheck.html>.
+
+- Check a table:
+
+`mysqlcheck --check {{table}}`
+
+- Check a table and provide credentials to access it:
+
+`mysqlcheck --check {{table}} -u {{user}} -p`
```

Comment:
```suggestion
`mysqlcheck --check {{table}} --user {{username}} --password {{password}}`
```
I would suggest using the long option here too and having the placeholder for the password.